tests/test_interface/test_text_interface.py

```diff
@@ -55,6 +56,21 @@ def test_rotation(self):
         )
         assert text._create_qt_object().rotation() == 123
 
+    def test_transformOriginPoint(self):
```

Comment:
this name should be `test_transform_origin_point` (following typical [python naming conventions](https://namingconvention.org/python/), where snake case is used for most things)